Packs/ServiceNow/ReleaseNotes/1_1_1.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+- __README.md__
```

Comment:
We shouldn't have this entry. If sdk is enforcing this open a bug.

Reply:
It does not enforce it. changed.